### ISSUE TITLE
TextColumn::make('date') dateformatting date() only instead of dateTime()

### ISSUE THREAD

**Issue by eelco2k** [amendozaaguiar/filament-route-statistics#4]
The textColumn date contains also the time, but the time is always 00:00:00 which is useless space...

original:
```
 TextColumn::make('date')
                    ->label(__('filament-route-statistics::filament-route-statistics.table.columns.date'))
                    ->dateTime()
                    ->searchable()
                    ->sortable(),
```

without time 00:00:00 ...

```
 TextColumn::make('date')
                    ->label(__('filament-route-statistics::filament-route-statistics.table.columns.date'))
                    ->date()
                    ->searchable()
                    ->sortable(),
```

**Comment by amendozaaguiar** [comment]
Hello, I've just reviewed your request. The Laravel package [laravel-route-statistics](https://github.com/bilfeldt/laravel-route-statistics/blob/main/config/route-statistics.php) has a configuration option called "aggregate." This setting controls how we should aggregate the requests. The possible values are: MINUTE, HOUR, DAY, MONTH, YEAR. I will look for a solution to apply to the available types of values in the configuration of laravel-route-statistics.

**Comment by amendozaaguiar** [comment]
Fix in commit [511dde1](https://github.com/amendozaaguiar/filament-route-statistics/commit/511dde186f449b2fe8feee4a1a64070dae817715) [v2.4.0](https://github.com/amendozaaguiar/filament-route-statistics/releases/tag/2.4.0)

Add in resource config date_format 
```
 'resource' => [
        'slug' => null,
        'navigation_icon' => 'heroicon-o-chart-bar-square',
        'date_format' => 'M j, Y H:i:s',
    ]
```
